src/nnet3bin/nnet3-combine.cc

```diff
@@ -30,33 +84,39 @@ int main(int argc, char *argv[]) {
     typedef kaldi::int64 int64;
 
     const char *usage =
-        "Using a subset of training or held-out examples, compute an optimal combination of a\n"
-        "number of nnet3 neural nets by maximizing the objective function.  See documentation of\n"
-        "options for more details.  Inputs and outputs are 'raw' nnets.\n"
+        "Using a subset of training or held-out examples, compute the average\n"
+        "over the first n nnet3 models where we maxize the objective function\n"
+        "for n. Note that the order of models has been reversed before\n"
+        "being fed into this binary. So we are actually combining last n models.\n"
+        "Inputs and outputs are 'raw' nnets.\n"
         "\n"
-        "Usage:  nnet3-combine [options] <nnet-in1> <nnet-in2> ... <nnet-inN> <valid-examples-in> <nnet-out>\n"
+        "Usage:  nnet3-combine [options] <nnet-in1> <nnet-in2> ... <nnet-inN> <valid-example-in> <nnet-out>\n"
```

Comment:
i think 'examples' should actually be 'examples'-- or just 'egs' would be fine.  because it's an archive of multiple egs.